api-tests/src/test/java/org/openmrs/module/reporting/data/encounter/evaluator/EncountersForPatientDataEvaluatorTest.java

```diff
@@ -52,7 +55,13 @@ public void shouldReturnEncountersInActiveVisit() throws Exception {
         Context.getLocationService().saveLocation(location);
 
         EvaluationContext context = new EvaluationContext();
-        Patient patient = data.randomPatient().save();
+        PatientBuilder patientBuilder = data.randomPatient();
```

Comment:
This was not a consistent problem, but because `randomPatient()` generates an age between 1 and 90, it's not relatively easy to generate a patient whose birthday is after the date of the visit in the database (2013-10-01), which doesn't work. This change ensures that the patient was at least born before 2013-10-01.

Reply:
I'm not sure I understand, as this patient shouldn't have any visits in the database yet.  But this is fine with me.  Something probably worth adding as a feature to PatientBuilder (to be able to specify a min/max birthdate) but not going to worry about that for now.  If this becomes a recurring need, we should do that.

Reply:
Yeah, there's no visit in the database, but the visits created below are setup for 2013-10-01. Maybe in the long run, it's more flexible just to futz with some date math to produce visists at the appropriate date?